### PR TITLE
Enforcing runtimeClassName

### PR DESCRIPTION
**Pull request by jesmg**
Added "spec.containers.enforcing.runtimeClassName" in Space object. 
Setting this property will ensure the given runtimeClassName is used to run all the pods in the Space.

This PR solves https://github.com/k8spin/k8spin-operator/issues/10.

### CODE REVIEW

File: deployments/kubernetes/operator.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@ spec:
       containers:
       - name: k8spin-operator
         image: ghcr.io/k8spin/k8spin-operator:latest
+        imagePullPolicy: IfNotPresent
```
Comment (angelbarrera92) [inline deployments/kubernetes/operator.yaml:23]: Thanks!

**Comment by angelbarrera92** [comment]
Hi @jesmg!!! Amazing contribution mate!

Why did you decide to use the `enforcing` attribute in the CRD to add the runtimeClassName? @paurosello what do you think about the `enforcing` word?

Apart from the naming convention, I miss some tests and docs, but at the same time, we can contribute them in future releases.

Thanks!

**Comment by jesmg** [comment]
> Why did you decide to use the `enforcing` attribute in the CRD to add the runtimeClassName? @paurosello what do you think about the `enforcing` word?

Because we are **enforcing** a runtimeClassName, not setting a default value (all the containers are forced to run with the specified runtime).  I  could have added `runtimeClassName` directly under spec.containers, but I think we will want to add new "enforcing" options in the future.

**Comment by paurosello** [comment]
I am fine with setting it in a different attribute but I would set it to
"enforce" not "enforcing", otherwise seems good to me.

Missatge de Jesús ***@***.***> del dia dl., 5 d’abr. 2021 a
les 10:10:

**Comment by jesmg** [comment]
Done! :)

**Pull request merged**